api/conf/settings/logging.py

```diff
@@ -43,81 +41,92 @@ def health_check_filter(record: LogRecord) -> bool:
         },
     },
     "formatters": {
-        "django.server": {
-            "()": "django.utils.log.ServerFormatter",
-            "format": "[{server_time}] {message}",
+        "console": {
+            "format": "[{asctime} - {name} - {lineno:>3}][{levelname}] {message}",
             "style": "{",
         },
-        "console": {
-            "format": "[%(asctime)s - %(name)s - %(lineno)3d][%(levelname)s] [%(request_id)s] %(message)s",  # noqa: E501
+        "json": {
+            "()": structlog.stdlib.ProcessorFormatter,
+            "processor": structlog.processors.JSONRenderer(),
```

Comment:
Suggestion for how to handle changing the log format for local. Along with the suggestion below for these lines (and updating references to the `json` formatter to `structured`, we'd need to add:

```py
LOG_PROCESSOR = config("LOG_PROCESSOR", default="logfmt" if ENVIRONMENT == "local" else "json")
```

Which uses the right default per environment, and allows easily switching to json format locally if needed by setting `LOG_PROCESSOR=json` in the local .env file.

```suggestion
        "structured": {
            "()": structlog.stdlib.ProcessorFormatter,
            "processor": structlog.processors.JSONRenderer() if LOG_PROCESSOR == "json" else structlog.processors.LogfmtRenderer(),
```

Reply:
I have updated to use the `ConsoleRenderer` during development. I feel like that'll be helpful.

<img width="2556" alt="Screenshot 2024-05-07 at 5 45 36 AM" src="https://github.com/WordPress/openverse/assets/16580576/9e915752-e96a-4663-ba52-d72a8479d747">

Reply:
Perfect!